src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -224,11 +227,41 @@ def start_rwd(request, format=None):
     """
     user = request.user if request.user.is_authenticated() else None
     created = now()
-    location = request.data['location']
+    location = request.data.get('location')
+
+    def validate_location(loc):
+        if loc is None or type(loc) is not list or len(loc) is not 2:
+            return False
+        else:
+            [lat, lng] = loc
+            return type(lat) is float and type(lng) is float
```

Comment:
Should we also allow `int`? It's unlikely someone would send one, but might be difficult for them to debug if they ever do.

Reply:
Good point. Yeah, I think this should be number.

Reply:
Replaced these with checks for whether the values were numeric and were not booleans.